frontend/test/simulate/artifacts/ClusterMutation/create-with-illegal-value/expected-errors.txt

```diff
@@ -1 +1,2 @@
-InvalidRequestContent: properties.network.hostPrefix: Invalid value '30' for field 'hostPrefix' (must be at most 26)
+InvalidRequestContent: properties.network.hostPrefix: properties.network.hostPrefix: Invalid value: 30: must be less than or equal to 26
+InvalidRequestContent: properties.dns.baseDomainPrefix: properties.dns.baseDomainPrefix: Invalid value: "": (must be a valid DNS RFC 1035 label)
```

Comment:
interestingly, it seems that previously this error was not caught in ARO-HCP, but I believe it matches the annotations.